ui/src/environments/test-env-base.ts

```diff
@@ -13,11 +13,9 @@ export const testEnvironmentBase = {
   zendeskHelpCenterUrl: 'http://aousupporthelp.zendesk.com/hc',
   shibbolethUrl: 'http://mock-nih.dev.test.firecloud.org',
   shouldShowDisplayTag: true,
-  enablePublishedWorkspaces: true,
   inactivityTimeoutSeconds: 99999999999,
   inactivityWarningBeforeSeconds: 5 * 60,
-  enableHomepageRestyle: true,
+  enablePublishedWorkspaces: true,
```

Comment:
While you're touching this, I'd propose turning this off everywhere. We're very far away from flipping this flag on, and certainly the current incarnation is not something we're planning to launch.

Reply:
We could write a script to look periodically for stale flags that haven't changed in X days and are the same in all environments. You really have to stay on these or they gang up on you.